great_expectations/expectations/metrics/query_metrics/query_column.py

```diff
@@ -0,0 +1,57 @@
+from typing import Any, Dict, Optional, Union
+
+from great_expectations.execution_engine import (
+    ExecutionEngine,
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.execution_engine.execution_engine import MetricDomainTypes
+from great_expectations.expectations.metrics.import_manager import sa
+from great_expectations.expectations.metrics.metric_provider import metric_value
+from great_expectations.expectations.metrics.query_metric_provider import (
+    QueryMetricProvider,
+)
+
+
+class QueryColumn(QueryMetricProvider):
```

Comment:
What kind of a metric is this?  Is it a map metric?  Or is it an aggregate metric?  Or something entirely different?  What output is expected from this metric?  How would it behave if it is utilized by `ParameterBuilder`?  Thanks.

Reply:
@alexsherstinsky this can be a map metric or an aggregate metric depending on the query defined. Complex SQL logic (e.g. subqueries) and db aggregate functions are allowed.